js/setup.js

```diff
@@ -0,0 +1,56 @@
+'use strict';
+
+var randomElement = function (array) {
+  var rand = Math.floor(Math.random() * array.length);
+  return array[rand];
+};
+
+document.querySelector('.setup').classList.remove('hidden');
+
+var similarListElement = document.querySelector('.setup-similar-list');
+var similarWizardTemplate = document.querySelector('#similar-wizard-template').content.querySelector('.setup-similar-item');
+
+var WIZARD_FIRST_NAMES = ['Иван', 'Хуан Себастьян', 'Мария', 'Кристоф', 'Виктор', 'Юлия', 'Люпита', 'Вашингтон'];
+var WIZARD_LAST_NAMES = ['да Марья', 'Верон', 'Мирабелла', 'Вальц', 'Онопко', 'Топольницкая', 'Нионго', 'Ирвинг'];
+var COAT_COLORS = ['rgb(101, 137, 164)', 'rgb(241, 43, 107)', 'rgb(146, 100, 161)', 'rgb(56, 159, 117)', 'rgb(215, 210, 55)', 'rgb(0, 0, 0)'];
+var EYES_COLORS = ['black', 'red', 'blue', 'yellow', 'green'];
+
+var createWizardsArray = function (wizardsNumber) {
```

Comment:
```suggestion
var createWizards = function (wizardsNumber) {
```
лучше не использовать слова, которые означают тип данных в джава скрипте, тк бывает, что в функцию приедет другой тип данных, и будет неловко, в любом случае функции всегда более абстрактны, и могут работать с разными типами данных